app/CPReflectionService/CPReflectionService.m

```diff
@@ -61,4 +61,14 @@ - (void)allPods:(void (^ _Nonnull)(NSArray<NSString *> * _Nullable pods, NSError
   }];
 }
 
+- (void)updateSpecRepos:(void (^)(NSError * _Nullable))reply
+{
+  [RBObject performBlock:^{
+    [RBObjectFromString(@"Pod::App") update_spec_repos];
```

Comment:
Yeah, I'm not sure either, we've not come close to seeng whether that happen  ( I mean someone could hit `install` when it was parsing, but parsing takes milliseconds normally and it's not been noticeable )